src/GNDStk/gnds-keyword-meta.hpp

```diff
@@ -3,7 +3,7 @@
 // meta_t
 // -----------------------------------------------------------------------------
 
-template<class T>
+template<class RESULT>
```

Comment:
Need to figure out the filename for this one too. Perhaps we need to have a `keyword` namespace and inside of that have the `Meta_t` and `Child_t` classes.

Reply:
Originally, I did plan to put child-node names and metadata-key names into one namespace. After sifting through the entire set of available GNDS files, I discovered that there's a small bit of overlap between those names. So, to avoid having more than one object of the same name in the same namespace, we either need to put them in different namespaces, or to slightly change some names from what they are in the GNDS files.

Reply:
Oh, I just re-read this remark and realized that you're talking about the two classes, not the objects of those classes. Given that I have child and meta namespaces, I could put the classes themselves into those. I'm actually using "keyword" for something else. No worries, something reasonable can be figured out.